http.md

```diff
@@ -0,0 +1,301 @@
+# JSON RPC HTTP Methods
+
+searcher nodes accept HTTP requests using the [JSON-RPC 2.0](https://www.jsonrpc.org/specification) specification.
+
+## RPC HTTP Endpoint
+
+**Default port:** 8899 e.g. [http://localhost:8899](http://localhost:8899), [http://192.168.1.88:8899](http://192.168.1.88:8899)
```

Comment:
Is the port 8899 fine or should define something else?

Reply:
8899 is the default rpc port. it is also configurable

Reply:
i think it'll be port 80/443